assets/agenda/components/AgendaPreviewPlanning.tsx

```diff
@@ -14,93 +15,203 @@ interface IOwnProps {
+                    key={item.guid}
+                    item={item}
+                    plan={plan}
+                    wireItems={wireItems}
+                    actions={coverageActions}
+                    user={user}
+                    previewGroup={previewGroup}
+                    restrictCoverageInfo={restrictCoverageInfo}
+                    previewConfig={previewConfig}
+                />
+            );
+        }
+
+        return (
+            <React.Fragment>
+                {/* Current Planning Items */}
+                {!plan ? null : (
+                    <div className="agenda-planning__container info-box">
+                        <div className="info-box__content">
+                            <span className="info-box__label">{gettext('Planning Item')}</span>
                             <AgendaPreviewCoverages
-                                key={planningItem.guid}
+                                key={plan.guid}
                                 item={item}
-                                plan={planningItem}
+                                plan={plan}
                                 wireItems={wireItems}
                                 actions={coverageActions}
                                 user={user}
                                 previewGroup={previewGroup}
                                 previewConfig={previewConfig}
                             />
-                        ))}
+                        </div>
+                    </div>
+                )}
+                {!otherPlanningItems.length ? null : (
+                    <div className="agenda-planning__container info-box">
+                        <div className="info-box__content">
+                            <span className="info-box__label">
+                                {plan == null ? gettext('Planning Items') : gettext('Other Planning Items')}
+                            </span>
+                            {otherPlanningItems.map((planningItem) => (
+                                <AgendaPreviewCoverages
+                                    key={planningItem.guid}
+                                    item={item}
+                                    plan={planningItem}
+                                    wireItems={wireItems}
+                                    actions={coverageActions}
+                                    user={user}
+                                    previewGroup={previewGroup}
+                                    previewConfig={previewConfig}
+                                />
+                            ))}
+                        </div>
+                    </div>
+                )}
+                {/* Secondary Planning Items */}
+                <div className="agenda-planning__container info-box">
+                    <div className="info-box__content">
+                        <span className="info-box__label">
+                            {gettext('Secondary Planning Items')}
```

Comment:
I'm not sure if `Secondary Planning Items` would make sense to the average user. Maybe something like `Related Planning Items`?